addons/binding/org.openhab.binding.wink/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="wink"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="wink_hub_2">
+    	<label>Wink Hub 2</label>
+	</bridge-type>
+	<thing-type id="light_bulb">
+        <supported-bridge-type-refs>
+        	<bridge-type-ref id="wink_hub_2" />
+        </supported-bridge-type-refs>
+        <label>Light Bulb (Dimmable)</label>
+        <description>Controls dimmable light bulb</description>
+
+        <channels>
+            <channel id="lightlevel" typeId="lightDimmer" />
+            <channel id="lightstate" typeId="lightSwitch" />
+        </channels>
+        <config-description>
+            <parameter name="uuid" type="text" required="true">
+                <label>UUID</label>
+                <description>Unique Identifier assigned by Wink</description>
+            </parameter>
+        </config-description>
+	</thing-type>
+    <thing-type id="remote">
```

Comment:
This could be removed, I've a local branch that add support for the remote but I'm not done yet so let's remove this for now and I'll send this separately later (in another PR)

Reply:
Alright, sounds like a plan, to leave it out not and add it with a feature PR later.

Reply:
agreed.  We'll remove it for now.  There is some functionality being developed in another fork.  We can merge it later.